runner/docker/Dockerfile.live-base-comfyui

```diff
@@ -32,8 +38,23 @@ RUN pip install git+https://github.com/yondonfu/comfystream.git
 RUN git clone https://github.com/yondonfu/comfystream.git && \
     cd comfystream && \
     pip install -r requirements.txt && \
-    cp -r nodes/tensor_utils /comfyui/custom_nodes/ && \
-    cd ..
+    cp -r nodes/tensor_utils /comfyui/custom_nodes/ 
+
+# Install ComfyUI-SAM2-Realtime (https://github.com/pschroedl/ComfyUI-SAM2-Realtime.git)
+RUN cd /comfyui/custom_nodes && \
+    git clone https://github.com/pschroedl/ComfyUI-SAM2-Realtime.git && \
+    cd ComfyUI-SAM2-Realtime && \
+    pip install -r requirements.txt
+
+# Install ComfyUI-StreamDiffusion (https://github.com/pschroedl/ComfyUI-StreamDiffusion.git)
+RUN cd /comfyui/custom_nodes && \
+    git clone https://github.com/pschroedl/ComfyUI-StreamDiffusion.git && \
```

Comment:
But another thing, which can be done separately if you prefer, is that we're not pinning the version of any of these nodes. So it could happen that the owners do a breaking change in their repo and it breaks our setup without us changing anything.

I would consider doing [something like this](https://github.com/livepeer/ai-worker/blob/ca5d650f9ebb60ee4ff8525b4076d19ba93fd169/runner/docker/Dockerfile.live-base-liveportrait#L44-L47), like cloning the repo, cd'ing inside and then checking out a specific commit. For the liveportrait one I just picked up the hash of the latest commit, which you can check easily with a command like this: `git ls-remote https://github.com/warmshao/FasterLivePortrait.git HEAD` (just learned this from GPT lol pretty cool)

Reply:
That is a great idea - will update.

This will also allow us to validate changes to the more nodes easily without changing the version on the image that is in production.